server/core/basic_cluster.go

```diff
@@ -317,9 +393,43 @@ func (bc *BasicCluster) PreCheckPutRegion(region *RegionInfo) (*RegionInfo, erro
 func (bc *BasicCluster) PutRegion(region *RegionInfo) []*RegionInfo {
 	bc.Lock()
 	defer bc.Unlock()
+	if bc.SelectConf.NewRegionFirst && region.GetTimestamp() >= uint64(time.Now().Unix())-bc.SelectConf.TimeThreshold {
```

Comment:
It's too intrusive to maintain such additional information.

Reply:
So just add a new set and a switch, and get rid of everything else？

Reply:
My opinion is to try to integrate it into original logic well, without apparent decrease of performance(e.g. scan regions frequently).

Reply:
I see. I will try to reduce the cost of the new scheme and get close to the desired effect.